src/client/imagesource.cpp

```diff
@@ -913,6 +913,9 @@ static void imageTransform(u32 transform, video::IImage *src, video::IImage *dst
 	else if (transform == 7)    // flip y then rotate by 90 degrees ccw
 		sxn = 3, syn = 1;  //   sx = (H-1) - dy, sy = (W-1) - dx
 
+	if (sxn == 0 && syn == 2)
+		return; // nothing to do
```

Comment:
Shouldn't the image be copied here? Are you sure that `dst` has the same pixel values as `src`?

Reply:
you're right, this makes no sense.